improver/calibration/ensemble_calibration.py

```diff
@@ -1003,25 +997,21 @@ def _calculate_location_parameter_from_mean(self, optimised_coeffs):
         # Calculate location parameter = a + b*X, where X is the
         # raw ensemble mean. In this case, b = beta.
         location_parameter = (
-            optimised_coeffs["alpha"]
-            + optimised_coeffs["beta"] * forecast_predictor.data
+            self.coefficients_cubelist.extract_strict("emos_coefficient_alpha").data
+            + self.coefficients_cubelist.extract_strict("emos_coefficient_beta").data
+            * forecast_predictor.data
         ).astype(np.float32)
 
         return location_parameter
 
-    def _calculate_location_parameter_from_realizations(self, optimised_coeffs):
+    def _calculate_location_parameter_from_realizations(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method name "_calculate_location_parameter_from_realizations" doesn't conform to '((test_[a-zA-Z0-9_]+)|([a-z_][a-z0-9_]{2,30}))$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5439763)